docs/en/reference/query-builder.rst

```diff
@@ -505,7 +505,7 @@ using ``addCriteria``:
     // ...
 
     $criteria = Criteria::create()
-        ->orderBy(['firstName', 'ASC']);
+        ->orderBy(['firstName' => 'ASC']);
```

Comment:
Can you please use `Criteria::ASC` here?

Reply:
Sure, gimme a moment though as now I need to clone :D

Reply:
Edit your branch from the online editor :D

Reply:
And clone later because of squash request? :P

Reply:
What squash request? I'm not THAT mad.

Reply:
I'd normally ask for one ;)